src/Kestrel.Core/Internal/Http2/Http2PeerSettings.cs

```diff
@@ -7,15 +7,20 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2
 {
     public class Http2PeerSettings
     {
-        // Note these are protocol defaults, not Kestrel defaults.
+        // Note these are protocol defaults
         public const uint DefaultHeaderTableSize = 4096;
         public const bool DefaultEnablePush = true;
         public const uint DefaultMaxConcurrentStreams = uint.MaxValue;
         public const uint DefaultInitialWindowSize = 65535;
-        public const uint DefaultMaxFrameSize = Http2Limits.MinAllowedMaxFrameSize;
+        public const uint DefaultMaxFrameSize = MinAllowedMaxFrameSize;
         public const uint DefaultMaxHeaderListSize = uint.MaxValue;
         public const uint MaxWindowSize = int.MaxValue;
 
+        // These are limits defined by the RFC https://tools.ietf.org/html/rfc7540#section-4.2, not Kestrel defaults.
```

Comment:
This comment is redundant with the one above. Remove it and the empty line above?